src/pages/profile/ProfileTab.css

```diff
@@ -1,3 +1,22 @@
 .picture {
   font-size: 128px;
+  margin: auto;
+  margin-top: 10%;
+  text-align: center;
 }
+.notLogged {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+
+  color: brown;
+}
+
+.editButton {
+  margin: auto;
+  margin-top: 10%;
+  margin-bottom: 10%;
+  text-align: center;
+}
```

Comment:
.editButton benutzt du um die Buttons zu centern? Das macht man mit flex oder mit grid, wenn zwei Items nebeneinander liegen sollen, nicht mit margin, weil du sonst bei verschiedenen Displaygrößen Probleme bekommen kannst. Für Grid gibt es auch Components in Ionic die das Styling schon haben.